sdk/python/feast/infra/offline_stores/bigquery_source.py

```diff
@@ -78,19 +91,14 @@ def __init__(
 
         super().__init__(
             _name if _name else "",
-            event_timestamp_column,
-            created_timestamp_column,
-            field_mapping,
-            date_partition_column,
```

Comment:
btw you can remove `date_partition_column` in `to_proto` and `from_proto`

Reply:
done